airflow/utils/log/file_task_handler.py

```diff
@@ -492,10 +492,10 @@ def _init_file(self, ti):
     def _read_from_local(worker_log_path: Path) -> tuple[list[str], list[str]]:
         messages = []
         logs = []
-        files = list(worker_log_path.parent.glob(worker_log_path.name + "*"))
+        files = sorted(worker_log_path.parent.glob(worker_log_path.name + "*"))
         if files:
-            messages.extend(["Found local files:", *[f"  * {x}" for x in sorted(files)]])
-        for file in sorted(files):
+            messages.extend(["Found local files:", *[f"  * {x}" for x in files]])
```

Comment:
Same as above, it’s even better to split the first item into its own `append` to save building the list.